### DISCUSSION TITLE
ISO backup doesn't work in new verison?

### DISCUSSION THREAD

**Discussion by joelang1699** [automatic-ripping-machine/automatic-ripping-machine#1045] [Q&A]
Hello Guys

About two years ago I was using ARM in a VM to backup DVDs to ISO files which worked wonderfully, until I dleeted the VM by accident that is lol.

I'm using ARM (2.6.67) in a docker now in yet MakeMKV will only now rip to MKV, surely I can set this to rip to ISO as before?

Additionally, I installed MakeMKV standalone in docker on the same machine which **can** rip straight to ISO.

I'm obviously missing something here??

**Comment by joelang1699** [comment]
Here's MakeMKV standalone, same system/drive, yet I still can't get MakeMKV in ARM to do the same.

![Screenshot 2024-01-25 135902](https://github.com/automatic-ripping-machine/automatic-ripping-machine/assets/13611738/d3cd9d15-b98c-49f5-9d98-f5a6cdcc8c01)
![Screenshot 2024-01-25 141319](https://github.com/automatic-ripping-machine/automatic-ripping-machine/assets/13611738/f6ceb6c6-6d8d-4b27-84e5-1b29d87f359d)

**Comment by joelang1699** [comment]
Can anyone shed some light on this? I really need to be able to rip to ISO as per MakeMKV standalone.

**Comment by MrJay84** [comment]
I have the same issue.
